Std/Data/Array/Basic.lean

```diff
@@ -36,6 +36,18 @@ Sort an array using `compare` to compare elements.
 def qsortOrd [ord : Ord α] (xs : Array α) : Array α :=
   xs.qsort λ x y => compare x y |>.isLT
 
+set_option linter.unusedVariables.funArgs false in
+/--
+Returns the first minimal element among `d` and elements of the array.
+If `start` and `stop` are given, only the subarray `xs[start:stop]` is
+considered (in addition to `d`).
+-/
+@[inline]
+protected def minWith [ord : Ord α]
+    (xs : Array α) (d : α) (start := 0) (stop := xs.size) : α :=
+  xs.foldl (init := d) (start := start) (stop := stop) λ min x =>
```

Comment:
```suggestion
  xs.foldl (init := d) (start := start) (stop := stop) fun min x =>
```

Reply:
Looks like someone cleaned this up in the mean time :) Done.